libse/Forms/FixCommonErrors/FixDanishLetterI.cs

```diff
@@ -17,626 +17,19 @@ public void Fix(Subtitle subtitle, IFixCallbacks callbacks)
-                                 MyRegEx(@"\ber i allesammen\b"),
-                                 MyRegEx(@"\bEr i allesammen\b"),
-                                 MyRegEx(@"\ber i der\b"),
-                                 MyRegEx(@"\bEr i der\b"),
-                                 MyRegEx(@"\bEr i fra\b"),
-                                 MyRegEx(@"\bEr i gennem\b"),
-                                 MyRegEx(@"\ber i gennem\b"),
-                                 MyRegEx(@"\ber i glade\b"),
-                                 MyRegEx(@"\bEr i glade\b"),
-                                 MyRegEx(@"\bEr i gået\b"),
-                                 MyRegEx(@"\ber i gået\b"),
-                                 MyRegEx(@"\ber i her\b"),
-                                 MyRegEx(@"\bEr i her\b"),
-                                 MyRegEx(@"\ber i imod\b"),
-                                 MyRegEx(@"\bEr i imod\b"),
-                                 MyRegEx(@"\ber i klar\b"),
-                                 MyRegEx(@"\bEr i klar\b"),
-                                 MyRegEx(@"\bEr i mætte\b"),
-                                 MyRegEx(@"\ber i mætte\b"),
-                                 MyRegEx(@"\bEr i med\b"),
-                                 MyRegEx(@"\ber i med\b"),
-                                 MyRegEx(@"\ber i mod\b"),
-                                 MyRegEx(@"\bEr i mod\b"),
-                                 MyRegEx(@"\ber i okay\b"),
-                                 MyRegEx(@"\bEr i okay\b"),
-                                 MyRegEx(@"\ber i på\b"),
-                                 MyRegEx(@"\bEr i på\b"),
-                                 MyRegEx(@"\bEr i parate\b"),
-                                 MyRegEx(@"\ber i parate\b"),
-                                 MyRegEx(@"\ber i sikker\b"),
-                                 MyRegEx(@"\bEr i sikker\b"),
-                                 MyRegEx(@"\bEr i sikre\b"),
-                                 MyRegEx(@"\ber i sikre\b"),
-                                 MyRegEx(@"\ber i skøre\b"),
-                                 MyRegEx(@"\bEr i skøre\b"),
-                                 MyRegEx(@"\ber i stadig\b"),
-                                 MyRegEx(@"\bEr i stadig\b"),
-                                 MyRegEx(@"\bEr i sultne\b"),
-                                 MyRegEx(@"\ber i sultne\b"),
-                                 MyRegEx(@"\bEr i tilfredse\b"),
-                                 MyRegEx(@"\ber i tilfredse\b"),
-                                 MyRegEx(@"\bEr i to\b"),
-                                 MyRegEx(@"\ber i ved at\b"),
-                                 MyRegEx(@"\ber i virkelig\b"),
-                                 MyRegEx(@"\bEr i virkelig\b"),
-                                 MyRegEx(@"\bEr i vågne\b"),
-                                 MyRegEx(@"\ber i vågne\b"),
-                                 MyRegEx(@"\bfanden vil i?"),
```

Comment:
It should be the ending of a question like "What do you want?"... so it looks like a bug.